CHANGELOG.md

```diff
@@ -3,7 +3,11 @@
 The changelog for **SwifterSwift**. Also see the [releases](https://github.com/SwifterSwift/SwifterSwift/releases) on GitHub.
 
 ## Upcoming Release
+
```

Comment:
As I mentioned before, look at the current style and match it. You've added a new empty line here.